src/pages/projectView/projectActivityPanel/screens/rewardSelection/FundingFormRewards.tsx

```diff
@@ -24,6 +27,8 @@ export const FundingFormRewards = ({ readOnly, onRewardClick }: Props) => {
 
   const hasRewards = rewards && rewards.length
 
+  const isProjectPage = location.pathname.includes(PathName.project)
```

Comment:
Not too relevant for this PR, but would this type of check be worth having in a global state instead?

Reply:
That's a great catch, and yes we actually already have a pattern for knowing whether or not you're on a particular page or pages, with Jotai atoms. I've used it extensively around the TopNavBar.
It does have some boiler plate, so didn't go there for the hot-fix. Will take a note of it and change it to use the global way. 
I'm also rethinking, how entries are detached from the project, and we can make the entry experience smoother. 
The gist of the idea is:
- move entries to be inside project both interms of routes `../entry/id` -> `.../project/id/entry/id` and in terms of code organization.
- That way there'll be a single context for both entry and project, while currently we have different ones, and going from entry to a project project, would mean having to deal with different states. 

Will make a plan for this and setup a call this week to go over it.